docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
     #     source: ./packages/greencheck
```

Comment:
Should we keep this file?

Reply:
I think we were using docker-compose because cloud build needed us to build a series of separate containers, or by separating them into separate services we might be able scale them independently.

Given that travis already has the various bits of software like PG, MySQL, Redis and so installed. 

I think we can remove it, as we are no longer using the older versions of PHP.